### PR TITLE
reduced number of declared variables

### PR DESCRIPTION
**Pull request by jaykay99992**
I combined the following variables in one array passed to createRadialGradient() instead of passing each variable independently.      
      
      double cx,
      double cy,
      double fx,
      double fy,
      double r,
      
      
      To facilitate the management of parameters.

### CODE REVIEW

File: jhotdraw-samples/jhotdraw-samples-misc/src/main/java/org/jhotdraw/samples/svg/io/SVGOutputFormat.java [hunk 1]
```diff
@@ -7,6 +7,21 @@
...
+import static org.jhotdraw.draw.AttributeKeys.FONT_UNDERLINE;
+import static org.jhotdraw.draw.AttributeKeys.STROKE_CAP;
+import static org.jhotdraw.draw.AttributeKeys.STROKE_COLOR;
+import static org.jhotdraw.draw.AttributeKeys.STROKE_DASHES;
+import static org.jhotdraw.draw.AttributeKeys.STROKE_DASH_PHASE;
+import static org.jhotdraw.draw.AttributeKeys.STROKE_JOIN;
+import static org.jhotdraw.draw.AttributeKeys.STROKE_MITER_LIMIT;
+import static org.jhotdraw.draw.AttributeKeys.STROKE_WIDTH;
+import static org.jhotdraw.draw.AttributeKeys.TRANSFORM;
+import static org.jhotdraw.draw.AttributeKeys.WINDING_RULE;
```
Comment (Tiruakhila) [inline jhotdraw-samples/jhotdraw-samples-misc/src/main/java/org/jhotdraw/samples/svg/io/SVGOutputFormat.java]: Did you add the imports by mistake or on a purpose, what is the reason to add it as this was never in the scope.

**Comment by jaykay99992** [comment]
@Tiruakhila These imported variables are actually used in lines below for example :

`    writeAttribute(elem, "stroke-miterlimit", STROKE_MITER_LIMIT.get(m), 4d);`

But I just realized that it is just an extra import since everything from JhotDrawCore is already imported in line 51 :

import org.jhotdraw.draw.*;

So they are actually useless in this case. Do i remove them and proceed to merge ?

**Comment by Tiruakhila** [comment]
Yes, please remove as they are unnecessary variables. Once you change it please let me know, so that I can approve the Pull Request.

**Comment by jaykay99992** [comment]
@Tiruakhila I just updated the code and removed the extra imports, do I merge it like this ?

**Comment by Tiruakhila** [comment]
Yes please.

**Review by Tiruakhila — APPROVED** [review]

**Comment by jaykay9999** [comment]
Code approved and merged

**Pull request merged**